products/opensuse/main.pm

```diff
@@ -499,7 +499,7 @@ sub load_consoletests() {
         {
             loadtest "console/glibc_i686.pm";
         }
-        if (!gnomestep_is_applicable) {
+        if (get_var("DESKTOP") !~ /gnome|kde/) {
```

Comment:
the decision why for gnome and kde different reasons apply is not obvious to anyone reading this so please improve that with a proper decision helper sub or at least a better comment